### ISSUE TITLE
Grip a Next.js app 

### ISSUE THREAD

**Issue by iskandar47** [stakeordie/griptape.js#174]
is there a way to grip a next.js app ?

**Comment by DC1991Lau** [comment]
Hello, is there any developments regarding this issue?

**Comment by DC1991Lau** [comment]
There is any workaround to use it in a Nextjs App?

**Comment by iskandar47** [comment]
> There is any workaround to use it in a Nextjs App?

yes i think there is, one member from the team told me once that this is possible and it require a lot of configuration . i have no idea what this configuration is
